zerver/migrations/0219_alter_field_realm_video_chat_provider.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-05-09 06:54
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+def update_existing_video_chat_provider_values(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model('zerver', 'Realm')
+    all_realms = Realm.objects.all()
+
+    for realm in all_realms:
+        realm.video_chat_provider = Realm.VIDEO_CHAT_PROVIDERS.index(Realm.video_chat_provider)
+        realm.save(update_fields=["video_chat_provider"])
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0218_remove_create_stream_by_admins_only'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='realm',
+            name='video_chat_provider',
+            field=models.PositiveSmallIntegerField(default=0),
+        ),
+        migrations.RunPython(update_existing_video_chat_provider_values,
+                             reverse_code=migrations.RunPython.noop),
```

Comment:
You should write a reverse migration for this, since otherwise the state will be busted if we have to roll back this migration (everyone will have been reset to Jitsi).